python/adelphi/tests/unit/test_anonymize.py

```diff
@@ -6,7 +6,7 @@
 	FIELD_PREFIX,\
 	INDEX_PREFIX,\
 	get_name
-from schema_util import get_schema
+from tests.unit.schema_util import get_schema
```

Comment:
This + the addition of an __init__.py for tests were necessary to facilitate test discovery by pytest